Resources/Storage/SRM2Storage.py

```diff
@@ -899,7 +899,12 @@ def __getFile( self, src_url, dest_file ):
       self.log.debug( "__getFile: Local file already exists %s. Removing..." % dest_file )
       os.remove( dest_file )
     srctype = self.defaulttype
-    src_spacetokendesc = self.spaceToken
+    # lcg-cp download from a bestman2 SE does not work when src_spacetokendesc specified
+    backendType = gConfig.getValue( '/Resources/StorageElements/%s/BackendType' % ( self.name ), '' )
```

Comment:
Thanks @chaen, I have updated my code.
Thanks @phicharp, good point. I will reconfirm the behavior.